vowpalwabbit/slim/test/ut_explore.cc

```diff
@@ -9,7 +9,7 @@
 
 using namespace ::testing;
 
-TEST(ExploreTestSuite, sampling_rank)
+TEST(explore_slim, sampling_rank)
```

Comment:
If you change this you also need to update this: https://github.com/VowpalWabbit/vowpal_wabbit/blob/5c920b44f3f554820ff1fce01a7976dd1907814f/.github/workflows/build_vw_slim.yml#L31